### ISSUE TITLE
Support date in MM/DD/YY format

### ISSUE THREAD

**Issue by familywiser1** [GetStream/stream-chat-react-native#121]
**Describe the bug**
Channel Preview currently shows date in `DD/MM/YY` format. unfortunately, this date format does not work for US based users. 

**Dev environment info (please complete/provide the following information):**

- Package version (stream-chat-react-native/stream-chat-expo): stream-chat-expo
- react-native or expo version: expo
- Device/Emulator (android/ios) and OS version: all

**To Reproduce**
Follow react-native tutorials at https://getstream.io/chat/react-native-chat/tutorial/#custom-preview . I think the problem is at https://github.com/GetStream/stream-chat-react-native/blob/master/src/components/ChannelPreview.js#L89

**Expected behavior**
Allow clients to specify the date format.

**Comment by vishalnarkhede** [comment]
@familywiser1 thanks for brining this up. We will add a props to customize date format!!

**Comment by vishalnarkhede** [comment]
@familywiser1 this has been published - [v0.6.3](https://github.com/GetStream/stream-chat-react-native/releases/tag/v0.6.3)

You can use the prop `formatLatestMessageDate` in `ChannelPreviewMessenger` component.

Please let us know if it works for you :)

**Comment by familywiser1** [comment]
@vishalnarkhede  - Sorry, this may be a dumb question but how do I pass props to ChannelPreviewMessenger?

This is how I am using `ChannelList`

```
<ChannelList
                filters={filters}
                sort={sort}
                options={options}
 Preview={ChannelPreviewMessenger}/>
```

**Comment by familywiser1** [comment]
> @familywiser1 this has been published - [v0.6.3](https://github.com/GetStream/stream-chat-react-native/releases/tag/v0.6.3)
> 
> You can use the prop `formatLatestMessageDate` in `ChannelPreviewMessenger` component.
> 
> Please let us know if it works for you :)

Hi @vishalnarkhede  - Thanks so much for fixing it so quickly. I will really appreciate if you can let me know how to pass props to `ChannelPreviewMessenger`. This is how I am using ChannelList

```
<ChannelList
                filters={filters}
                sort={sort}
                options={options}
 Preview={ChannelPreviewMessenger}/>
```

I am not how to pass custom props to `ChannelPreviewMessenger`. Any code example or link to documentation would be really helpful

**Comment by type** [comment]
@familywiser1 Here's what I suggest trying:

```
<ChannelList
    filters={filters}
    sort={sort}
    options={options}
    Preview={(props) => <ChannelPreviewMessenger
                 formatLatestMessageDate={(date) => moment(date).toISOString()}
                {...props}
               />
    }
/>
```
